pkg/ingester/ingester_v2.go

```diff
@@ -271,7 +319,14 @@ func (i *Ingester) stoppingV2(_ error) error {
 	}
 
 	// Next initiate our graceful exit from the ring.
-	return services.StopAndAwaitTerminated(context.Background(), i.lifecycler)
+	if err := services.StopAndAwaitTerminated(context.Background(), i.lifecycler); err != nil {
+		level.Warn(util.Logger).Log("msg", "stopping ingester lifecycler", "err", err)
```

Comment:
Just to me more clear.
```suggestion
		level.Warn(util.Logger).Log("msg", "failed to stop ingester lifecycler", "err", err)
```